scripts/delayed.js

```diff
@@ -1,7 +1,64 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './lib-franklin.js';
+import { sampleRUM, getMetadata } from './lib-franklin.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+
+function loadGoogleTagManager() {
+  // Initialize the data layer
+  /* global userdata */
+  /* eslint no-undef: "error" */
+  const locale = getMetadata('locale');
+  const pageCountry = {
+    en: 'INT',
+    de: 'DE',
+  };
+  const conf = {
+    pageArea: 'web',
+    pageCountry: pageCountry[locale],
+    pageEnvironment: 'prod_publish',
```

Comment:
Would this vary by environment? For e.g. should it be set differently for localhost/preview/live ? If yes, we can perhaps derive from the page's url.

Reply:
Good point, addressed in 6bb1fbe